### PR TITLE
added sessionSigs and validators where needed

### PR DESCRIPTION
**Pull request by glitch003**

### CODE REVIEW

File: packages/encryption/src/lib/params-validators.ts
Comment (Adarsh-Kumar28) [inline packages/encryption/src/lib/params-validators.ts:1]: Should we add a check for the case when both the `authSig` & `sessionSigs` are provided?

File: packages/misc/src/lib/misc.ts [hunk 1]
```diff
@@ -204,14 +202,13 @@ export const checkType = ({
  * @returns { boolean }
```
Comment (Adarsh-Kumar28) [inline packages/misc/src/lib/misc.ts:202]: In the comments above:

`* @param { object } authSig` -> `* @param { JsonAuthSig } authSig`

**Review by Adarsh-Kumar28 — COMMENTED** [review]
Looks good, left a comment. There are a few places where the sessionSigs are still missing. I'll add a commit to this PR addressing that.

File: packages/encryption/src/lib/params-validators.ts
Comment (Adarsh-Kumar28) [reply packages/encryption/src/lib/params-validators.ts:1]: Addressed this: [feat: Validate if only either authSig or sessionSigs are provided](https://github.com/LIT-Protocol/js-sdk/pull/54/commits/757aa0732f17ab8797936f06d6d2fce560c493cb)

**Comment by Ansonhkg** [comment]
added a little tool to check if the built package dependencies matches the package version. This **does not** affect the current workflow. 

```
yarn build:verify
```

**Comment by Ansonhkg** [comment]
@Adarsh-Kumar28 we can merge this

**Pull request merged**